otx/algorithms/segmentation/adapters/mmseg/models/backbones/litehrnet.py

```diff
@@ -115,6 +124,8 @@ def forward(self, x):
 
 
 class CrossResolutionWeighting(nn.Module):
+    """Cross resolution weighting."""
+
     def __init__(
         self, channels, ratio=16, conv_cfg=None, norm_cfg=None, act_cfg=(dict(type="ReLU"), dict(type="Sigmoid"))
```

Comment:
pylint does not recommend declaring a dict or list as the default in an argument. (dangerous-default-value)
This can be modified as shown below.
```
def __init__(
        self, channels, ratio=16, conv_cfg=None, norm_cfg=None, act_cfg=None
    ):
    act_cfg = act_cfg if act_cfg else (dict(type="ReLU"), dict(type="Sigmoid"))
```

Reply:
Revised. Thanks:)